client/state/data-layer/wpcom/sites/plugins/new/index.js

```diff
@@ -21,6 +28,38 @@ export const uploadPlugin = ( { dispatch }, action ) => {
 	}, action ) );
 };
 
+const showSuccessNotice = ( dispatch, { name } ) => {
+	dispatch( successNotice(
+		translate( "You've successfully uploaded the %(name)s plugin.", {
+			args: { name }
+		} ),
+		{ duration: 5000 }
+	) );
+};
+
+const showErrorNotice = ( dispatch, error ) => {
+	const knownErrors = {
+		exists: translate( 'This plugin is already installed on your site.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Plugin is already installed.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).